### ISSUE TITLE
draw series combination of solid and dashed line

### ISSUE THREAD

**Issue by piyushSinghalDemo** [apexcharts/vue-apexcharts#82]
Can we create line graph with combination of dashed and solid line as per x-axis zone wise, for exaample after 8 point there will dashed line, else solid line
![dashed_line](https://user-images.githubusercontent.com/30336741/55309754-56c5eb00-547c-11e9-9472-8968e4235390.png)

**Comment by junedchhipa** [comment]
Nope, not supported in the current version (3.6.5) of ApexCharts.
